Public/Src/Tools/SymbolDaemon/Program.cs

```diff
@@ -26,12 +32,38 @@ public static int Main(string[] args)
             {
                 Console.WriteLine(nameof(SymbolDaemon) + " started at " + DateTime.UtcNow);
                 Console.WriteLine(SymbolDaemon.SymbolDLogPrefix + "Command line arguments: ");
-                Console.WriteLine(string.Join(Environment.NewLine + SymbolDaemon.SymbolDLogPrefix, args));
+                Console.WriteLine(SymbolDaemon.SymbolDLogPrefix + string.Join(Environment.NewLine + SymbolDaemon.SymbolDLogPrefix, args));
                 Console.WriteLine();
 
                 SymbolDaemon.EnsureCommandsInitialized();
 
-                var confCommand = ServicePipDaemon.ServicePipDaemon.ParseArgs(args, new UnixParser());
+                // resolve response files
+                var resolvedArguments = args.SelectMany(arg =>
+                    {
+                        if (arg.StartsWith(ResponseFilePrefix))

```

Comment:
>ResponseFilePrefix [](start = 43, length = 18)

We have a logic for handling response file. See [http://index/?leftProject=BuildXL.ToolSupport&leftSymbol=gwduauxhomsh&file=CommandLineUtilities.cs&line=649](http://index/?leftProject=BuildXL.ToolSupport&leftSymbol=gwduauxhomsh&file=CommandLineUtilities.cs&line=649)

Reply:
discussed offline

---
In reply to: [336119672](https://github.com/microsoft/BuildXL/pull/1041#discussion_r336119672) [](ancestors = 336119672)